### DISCUSSION TITLE
Examples of valid input

### DISCUSSION THREAD

**Discussion by AndreyDmitriev39r** [jdwilkin4/contrast-ratio-repo#59] [Ideas]
Some users may not know every color format or may not remember some details of syntax.
What if we add somewhere examples of correct input for each format they can use:
```
RGB format : rgb (100, 1, 233)
HEX format: #AAAB1C
```
and other examples for other options.
I guess this stuff makes sense in how to use section

Maybe after this phrase:
`Enter a foreground and background color in RGB hexadecimal format (e.g., #000 or #FF00FF).`
we will place some interactive element with the text 'click here to see more examples', and on click it will show(somehow/somewhere?) list of such examples.

Or we can simply add list of examples in the text.

Would like some feed back beefore creating the issue.

**Comment by jdwilkin4** [comment]
I vote we just add a list of examples in the text for the how to section

**Reply by Kernix13** [reply]
I agree. I can see someone entering a foreground element in one syntax and the background in a different syntax. As if they have a pre-built site and they are copying over one of their 20 colors from their site.
